messages/es.json

```diff
@@ -63,6 +63,15 @@
   "pages.admin.redirects.table.status.inactive": "Inactiva",
   "pages.admin.redirects.table.status.title": "Status",
   "pages.admin.redirects.table.to": "Para",
+  "pages.admin.redirects.table.toolbar.export": "Exportar",
+  "pages.admin.redirects.table.toolbar.fields.hide-all": "Seleccionar campos visibles",
+  "pages.admin.redirects.table.toolbar.fields.label": "Ocultar todo",
+  "pages.admin.redirects.table.toolbar.fields.show-all": "Mostrar todo",
+  "pages.admin.redirects.table.toolbar.import": "Importar",
+  "pages.admin.redirects.table.toolbar.line-density.high": "Alta",
+  "pages.admin.redirects.table.toolbar.line-density.label": "Densidad de línea",
+  "pages.admin.redirects.table.toolbar.line-density.low": "Bajo",
```

Comment:
Since you're talking about "densidad", I believe that "Baja" would be the correct translation.